log_request_id/middleware.py

```diff
@@ -72,5 +72,6 @@ def _get_request_id(self, request):
 
         return self._generate_id()
 
-    def _generate_id(self):
+    @staticmethod
```

Comment:
It is better not to use `@staticmethod()`, because if you have to make this method back "non-static", there will be a bunch of refactoring.

```suggestion
    # noinspection PyMethodMayBeStatic
```

Reply:
All in good time